### ISSUE TITLE
"image-label" not snake case

### ISSUE THREAD

**Issue by constantinpape** [ome/ngff#109]
Most of the names in the spec are snake case. However, `image-label` is not (uses `-` instead of `_`), see https://ngff.openmicroscopy.org/latest/#label-md. Is there a particular reason for this?
If not, could it still be changed? I find it fairly confusing.
(But I understand that you might not want to do it for legacy reasons; I don't feel super strongly about it.)

**Comment by constantinpape** [comment]
Slightly related minor point: 
The text in https://ngff.openmicroscopy.org/latest/#label-md does not mention `version`, but the example json contains it.

**Comment by sbesson** [comment]
My assumption is that it's historical and according to https://github.com/ome/ngff/pull/70, if there was a change it should rather be `imageLabel` , `imageValue` .

This raises the question of how we want to review the keys introduced before the naming convention. Either supporting the legacy name and the new name for a while or making a clear breaking change both have merits

**Comment by constantinpape** [comment]
> My assumption is that it's historical and according to #70, if there was a change it should rather be `imageLabel` , `imageValue` .

You're right, we decided to go for camelCase. So indeed it should be `imageLabel` and `imageValue`.

> This raises the question of how we want to review the keys introduced before the naming convention. Either supporting the legacy name and the new name for a while or making a clear breaking change both have merits

My vote would be for a clear breaking change (which of course would mean a minor version bump).
But again, I don't have very strong opinions on this since I'm not the one who'll need to change the code for this ;).
